dart/lib/src/utils/add_tracing_headers_to_http_request.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:meta/meta.dart';
+
+import '../../sentry.dart';
+
+@internal
+void addTracingHeadersToHttpHeader(Map<String, dynamic> headers,
+    {ISentrySpan? span, Hub? hub}) {
```

Comment:
Since this is only called in ~~one~~ two places, and from there the hub is always non-null, we can just make hub non-null here and don't need the fallback to `Sentry.currentHub`.